falcon_dbapi/resources/elasticsearch.py

```diff
@@ -197,9 +199,12 @@ def _parse_tokens(self, obj_class, tokens, value, default_expression=None, preve
                 return query_method(self=obj_class, column_name=column_name, value=value,
                                     default_op='should' if tokens[-1] == 'or' else 'must')
             if column_name is not None:
-                if token not in self._underscore_operators:
+                if token not in chain(self._underscore_operators, sub_fields):
                     raise HTTPBadRequest('Invalid attribute', 'Param {} is invalid, part {} is expected to be a known '
                                                               'operator'.format('__'.join(tokens), token))
+                if token in sub_fields:
+                    column_name = ".".join([column_name, token])
```

Comment:
I think it should be more similar to support for nested fields, that is `column_name` is set from `accumulated` at the end of the loop.